modules/osd-create-cluster-ccs.adoc

```diff
@@ -0,0 +1,221 @@
+:osd-on-gcp:
+endif::[]
+
+:_content-type: PROCEDURE
+ifdef::osd-on-aws[]
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on AWS with CCS
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+[id="osd-create-gcp-cluster-ccs_{context}"]
+= Creating a cluster on GCP with CCS
+endif::osd-on-gcp[]
+
+By using the Customer Cloud Subscription (CCS) billing model, you can create an {product-title} cluster in an existing 
+ifdef::osd-on-aws[]
+{AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+{GCP} 
+endif::osd-on-gcp[]
+account that you own.
+
+You must meet several prerequisites if you use the CCS model to deploy and manage {product-title} into your AWS account.
+
+.Prerequisites
+
+ifdef::osd-on-aws[]
+* You have configured your AWS account for use with {product-title}.
+* You have not deployed any services in your AWS account.
+* You have configured the AWS account quotas and limits that are required to support the desired cluster size.
+* You have an `osdCcsAdmin` AWS Identity and Access Management (IAM) user with the `AdministratorAccess` policy attached.
+* You have set up a service control policy (SCP) in your AWS organization.
+* Consider having *Business Support* or higher from AWS.
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+* You have configured your GCP account for use with {product-title}.
+* You have configured the GCP account quotas and limits that are required to support the desired cluster size.
+* You have created a GCP project.
++
+[NOTE]
+====
+The project name must be 10 characters or less.
+====
+* You have enabled the Google Cloud Resource Manager API in your GCP project. For more information about enabling APIs for your project, see link:https://cloud.google.com/endpoints/docs/openapi/enable-api[the Google Cloud documentation].
+* You have an IAM service account in GCP called `osd-ccs-admin` with the following roles attached:
+  ** DNS Administrator
+  ** Organization Policy Viewer Owner
+  ** Project IAM Admin
+  ** Service Management Administrator
+  ** Service Usage Admin
+  ** Storage Admin
+* You have created a key for your `osd-ccs-admin` GCP service account and exported it to a file named `osServiceAccount.json`.
++
+[NOTE]
+====
+For more information about creating a key for your GCP service account and exporting it to a JSON file,  see link:https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys[Creating service account keys] in the Google Cloud documentation.
+====
+* Consider having *Production Support* or higher from GCP.
+* To prevent potential conflicts, consider having no other resources provisioned in the project prior to installing {product-title}.
+endif::osd-on-gcp[]
+
+.Procedure
+
+. Log in to {console-redhat-com} and click *Create Cluster*.
+
+. In the *Cloud* tab, click *Create Cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. In the *Billing model* dialog, configure the subscription type and infrastructure type:
+.. Select a subscription type.
```

Comment:
This could be a good place to link to the docs about subscriptions in the OCM docs, as once you select whether your cluster is On-Demand or Annual, you can't change it later: https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2022/html-single/managing_clusters/index#subscribing-osd-cluster_assembly-cluster-subscriptions